src/chapters/forms/form-submission-post.rst

```diff
@@ -0,0 +1,86 @@
+POST Form Submission
+====================
+
+
+Form Submission using POST
+--------------------------
+Instead of using GET and query parameters to submit form data, we can use POST.
+To submit a form using a POST request, set the ``method`` attribute to ``"POST"``.
+Form data submitted via POST will be contained in the HTTP message body. Using POST is a
```

Comment:
Technically query parameters are also encrypted in the HTTPS process.  What do you think about this? Data submitted by GET request is less secure than POST because GET request URLs
and the query action_page are cached and logged, possibly leaking sensitive data.

Reply:
Disregard the second comment. Your final sentence here closes the loop. I spoke too soon. It has a typo though: "Data submitted by GET requests..." (requests is plural)